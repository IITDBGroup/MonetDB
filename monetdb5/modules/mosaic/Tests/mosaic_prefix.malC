b:= bat.new(:int);
bat.append(b,50:int);
bat.append(b,51:int);
bat.append(b,51:int);
bat.append(b,51:int);
bat.append(b,51:int);
bat.append(b,51:int);
bat.append(b,51:int);
bat.append(b,60:int);
bat.append(b,67:int);
bat.append(b,67:int);
bat.append(b,67:int);
bat.append(b,67:int);
bat.append(b,67:int);
bat.append(b,67:int);
bat.append(b,67:int);
bat.append(b,255:int);
bat.append(b,256:int);
bat.append(b,256:int);
bat.append(b,256:int);
bat.append(b,256:int);
bat.append(b,256:int);
bat.append(b,255:int);
bat.append(b,255:int);
bat.append(b,255:int);
bat.append(b,255:int);
bat.append(b,255:int);

io.print(b);
mosaic.compress(b,"prefix");
b:= mosaic.decompress(b);
io.print(b);

c:= bat.new(:sht);
bat.append(c,50:sht);
bat.append(c,51:sht);
bat.append(c,51:sht);
bat.append(c,51:sht);
bat.append(c,51:sht);
bat.append(c,51:sht);
bat.append(c,51:sht);
bat.append(c,60:sht);
bat.append(c,67:sht);
bat.append(c,67:sht);
bat.append(c,67:sht);
bat.append(c,67:sht);
bat.append(c,67:sht);
bat.append(c,67:sht);
bat.append(c,67:sht);
bat.append(c,255:sht);
bat.append(c,256:sht);
bat.append(c,256:sht);
bat.append(c,256:sht);
bat.append(c,256:sht);
bat.append(c,256:sht);
bat.append(c,255:sht);
bat.append(c,255:sht);
bat.append(c,255:sht);
bat.append(c,255:sht);
bat.append(c,255:sht);

io.print(c);
mosaic.compress(c,"prefix");
c:= mosaic.decompress(c);
io.print(c);
