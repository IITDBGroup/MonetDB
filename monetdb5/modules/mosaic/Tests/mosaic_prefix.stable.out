stdout of test 'mosaic_prefix` in directory 'monetdb5/modules/mosaic` itself:


# 12:42:06 >  
# 12:42:06 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=32744" "--set" "mapi_usock=/var/tmp/mtest-6218/.s.monetdb.32744" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/mk/mosaic//Linux/var/MonetDB/mTests_monetdb5_modules_mal" "mosaic_prefix.mal"
# 12:42:06 >  

# MonetDB 5 server v11.20.0
# This is an unreleased version
# Serving database 'mTests_monetdb5_modules_mal', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs and 128bit integers dynamically linked
# Found 15.590 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2014 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.ins.cwi.nl:32744/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-6218/.s.monetdb.32744
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# MonetDB/R   module loaded

# 11:50:49 >  
# 11:50:49 >  "mclient" "-lmal" "-ftest" "-tnone" "-Eutf-8" "--host=/var/tmp/mtest-30332" "--port=39586"
# 11:50:49 >  

# 15:01:29 >  
# 15:01:29 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-15809" "--port=37660"
# 15:01:29 >  

#io.print(b);
#--------------------------#
# h	t  # name
# void	int  # type
#--------------------------#
[ 0@0,	50  ]
[ 1@0,	51  ]
[ 2@0,	51  ]
[ 3@0,	51  ]
[ 4@0,	51  ]
[ 5@0,	51  ]
[ 6@0,	51  ]
[ 7@0,	60  ]
[ 8@0,	67  ]
[ 9@0,	67  ]
[ 10@0,	67  ]
[ 11@0,	67  ]
[ 12@0,	67  ]
[ 13@0,	67  ]
[ 14@0,	67  ]
[ 15@0,	255  ]
[ 16@0,	256  ]
[ 17@0,	256  ]
[ 18@0,	256  ]
[ 19@0,	256  ]
[ 20@0,	256  ]
[ 21@0,	255  ]
[ 22@0,	255  ]
[ 23@0,	255  ]
[ 24@0,	255  ]
[ 25@0,	255  ]
#--------------------------#
# h	t  # name
# void	int  # type
#--------------------------#
[ 0@0,	50  ]
[ 1@0,	51  ]
[ 2@0,	51  ]
[ 3@0,	51  ]
[ 4@0,	51  ]
[ 5@0,	51  ]
[ 6@0,	51  ]
[ 7@0,	60  ]
[ 8@0,	67  ]
[ 9@0,	67  ]
[ 10@0,	67  ]
[ 11@0,	67  ]
[ 12@0,	67  ]
[ 13@0,	67  ]
[ 14@0,	67  ]
[ 15@0,	255  ]
[ 16@0,	256  ]
[ 17@0,	256  ]
[ 18@0,	256  ]
[ 19@0,	256  ]
[ 20@0,	256  ]
[ 21@0,	255  ]
[ 22@0,	255  ]
[ 23@0,	255  ]
[ 24@0,	255  ]
[ 25@0,	255  ]
#--------------------------#
# h	t  # name
# void	sht  # type
#--------------------------#
[ 0@0,	50  ]
[ 1@0,	51  ]
[ 2@0,	51  ]
[ 3@0,	51  ]
[ 4@0,	51  ]
[ 5@0,	51  ]
[ 6@0,	51  ]
[ 7@0,	60  ]
[ 8@0,	67  ]
[ 9@0,	67  ]
[ 10@0,	67  ]
[ 11@0,	67  ]
[ 12@0,	67  ]
[ 13@0,	67  ]
[ 14@0,	67  ]
[ 15@0,	255  ]
[ 16@0,	256  ]
[ 17@0,	256  ]
[ 18@0,	256  ]
[ 19@0,	256  ]
[ 20@0,	256  ]
[ 21@0,	255  ]
[ 22@0,	255  ]
[ 23@0,	255  ]
[ 24@0,	255  ]
[ 25@0,	255  ]
#--------------------------#
# h	t  # name
# void	sht  # type
#--------------------------#
[ 0@0,	50  ]
[ 1@0,	51  ]
[ 2@0,	51  ]
[ 3@0,	51  ]
[ 4@0,	51  ]
[ 5@0,	51  ]
[ 6@0,	51  ]
[ 7@0,	60  ]
[ 8@0,	67  ]
[ 9@0,	67  ]
[ 10@0,	67  ]
[ 11@0,	67  ]
[ 12@0,	67  ]
[ 13@0,	67  ]
[ 14@0,	67  ]
[ 15@0,	255  ]
[ 16@0,	256  ]
[ 17@0,	256  ]
[ 18@0,	256  ]
[ 19@0,	256  ]
[ 20@0,	256  ]
[ 21@0,	255  ]
[ 22@0,	255  ]
[ 23@0,	255  ]
[ 24@0,	255  ]
[ 25@0,	255  ]

# 12:42:06 >  
# 12:42:06 >  "Done."
# 12:42:06 >  

